features/test.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Tests projects using Tuist test
+  Scenario: The project is an application with tests (ios_app_with_tests)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture ios_app_with_tests into the working directory
+    Then tuist tests the project
+    Then tuist tests the scheme AppTests from the project
+    Then tuist tests the scheme App and configuration Debug from the project
```

Comment:
@fortmarek what do you think about having a fixture just for testing purposes, and have a target per platform. That way we can make sure that the testing command works fine for every platform: macOS, iOS, watchOS, tvOS.

Reply:
That's a good idea, let's do that 👍